scripts/jenkins/jenkins-job-cancel

```diff
@@ -42,7 +42,9 @@ def get_build_params(build):
             if elem.get('_class') == 'hudson.model.ParametersAction':
                 parameters = elem.get('parameters', {})
                 break
-        return set([(str(pair['name']), str(pair.get('value')),)
+
+        return set([(unicode(pair['name']),
+                     unicode(pair.get('value')),)
```

Comment:
Please use a variant that works across both python2 and 3 like `x.decode("utf-8")`.

Reply:
Good point, thank you. But I'm not trying to convert unicode into ascii. It's the other way around, which makes it a bit [trickier](https://anonbadger.wordpress.com/2016/01/05/python2-string-format-and-unicode/) with python 2.

I updated the code to be python 3 compatible and do the right thing.

Reply:
Oh, right, my confusion. Then you could have left out the conversion. It should already be type unicode. But it should work now either way.

Reply:
In python 3 this doesn't matter, you're right, but python 2 is not that smart.